<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_tageszettel</class>
 <widget class="QDialog" name="Dialog_tageszettel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_mitarb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>311</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_mitarb">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_import">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tagzet">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>601</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_kst">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tagzet_erstellen">
   <property name="geometry">
    <rect>
     <x>934</x>
     <y>370</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tageszettel erstellen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
